encoder/encoder.go

```diff
@@ -0,0 +1,19 @@
+package encoder
+
+import (
+	"github.com/smartcontractkit/ocr2keepers/pkg/types"
+)
+
+type Config struct{}
```

Comment:
This struct isn't actually being used in the encoder implementation yet, it's just here to satisfy the interface